apps/pie-docs/src/components/thumbnail/overview/overview.md

```diff
@@ -0,0 +1,307 @@
+    alt: "A thumbnail that displays placeholder image."
+} %}
+
+---
+
+## Dos and Don’ts
+
+{% usage {
+    do: {
+        type: usageTypes.text,
+        items: [
+          "Use to give visibility to content on other pages.",
+          "Use to show logos and photography."
+        ]
+    },
+    dont: {
+        type: usageTypes.text,
+        items: [
+            "Don’t insert images of users, if this is needed, use an [Avatar](/components/avatar/) instead.",
+            "Don’t use illustrations."
+        ]
+    }
+} %}
+
+---
+
+## Anatomy
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/anatomy.svg",
+    alt: "Thumbnail with two numbers specifying different parts of thumbnail anatomy.",
+    width: "218px",
+    variant: "secondary"
+} %}
+
+{% list {
+    type: listTypes.ordered,
+    items: [
+      "**Container:** To align the image in the center and highlight the white-space around the image.",
+      "**Image:** The fill can be an image, placeholder icon or a fill color."
+    ]
+} %}
+
+---
+
+## Variations
+
+### Default
+
+Use it for logos or as previews for images that lead to primary content.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/variant-default.svg",
+    alt: "A thumbnail component with a centred, small transparent image preview.",
+    width: "64px",
+    variant: 'secondary'
+} %}
+
+### Outline
+
+A border can be applied to ensure the container of the thumbnail is visible.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/variant-outline.svg",
+    alt: "A thumbnail placeholder with a thin border and a centred, small transparent image preview.",
+    width: "64px",
+    variant: "secondary"
+} %}
+
+---
+
+## Modifiers
+
+### Padding
+
+Adds extra space around the thumbnail, useful to center and add space around logos.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/modifier-padding.svg",
+    alt: "A thumbnail placeholder with additional space around an image preview.",
+    width: "64px",
+    variant: "secondary"
+} %}
+
+### Container fill
+
+The fill behind the image is set to $container-default by default, it can be changed as needed to any container colour token.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/thumbnail/container-fill.svg",
+    alt: "A thumbnail placeholder with a filled background behind image preview.",
+    width: "64px",
+    variant: "secondary"
+} %}
+
+---
+
+## Sizes
+
+Thumbnails have 3 main sizes to account for corner radius readability,XSmall, Small and Medium, each of them with maximum and minimum widths and heights to signal where it’s necessary to change into the variant with a different corner radius.
+
+The following table outlines the available Thumbnail sizes:
+
+{% notification {
+  type: "information",
+  message: "To calculate the border-radius of the inner container, subtract the padding (2px) from the thumbnail radius."
```

Comment:
I have no idea what this means – I'm guessing it's a design-specific info tip? 😆

Reply:
i think yea.. tho it feels like an implementation details and shouldn't be mentioned 👀

Reply:
removed